Doc/library/smtplib.rst

```diff
@@ -556,35 +556,32 @@ This example prompts the user for addresses needed in the message envelope ('To'
 and 'From' addresses), and the message to be delivered.  Note that the headers
 to be included with the message must be included in the message as entered; this
 example doesn't do any processing of the :rfc:`822` headers.  In particular, the
-'To' and 'From' addresses must be included in the message headers explicitly. ::
+'To' and 'From' addresses must be included in the message headers explicitly::
 
-   import smtplib
+    import smtplib
```

Comment:
Can you revert the indentation changes in this PR? Code examples have 3-space indents across the `Doc/` directory in CPython; changing this example to be indented by 4 spaces wouldn't be consistent.

Reply:
Oh ! I thought it was 4 for the inline code-blocks and 3 when using directives. My bad !